plugins/spammer/webapi.go

```diff
@@ -18,7 +18,9 @@ func handleRequest(c echo.Context) error {
 	switch request.Cmd {
 	case "start":
 		if request.Rate == 0 {
+			log.Infof("Requesting invalid spamming at rate 0 mps. Setting it to 1 mps")
 			request.Rate = 1
+			request.Unit = "mps"
```

Comment:
Not sure if needed here, as later mps is set as default in switch anyway. Also it still can be changed to mpm  if user provided it as a parameter. In log we can just say rate is set to 1 per time unit

Reply:
ok will remove this.